### ISSUE TITLE
Enablers problems with resources

### ISSUE THREAD

**Issue by icosac** [idra-lab/PLOP#2]
When checking if an action a<sub>i</sub> is an enabler of an action a<sub>j</sub>, with t(a<sub>i</sub>)<t(a<sub>j</sub>), we must ensure not only that the predicates match, but also that they are fully grounded. 

Consider the action a<sub>i</sub>=`grip_end(a1, b1)`, which deletes a predicate that must not be present for an action a<sub>j</sub>=`move_arm(a1, 1, 1, 1, 0, 0, 0)`, e.g., `gripping(a1, _)`. The predicate is not fully grounded as we do not care which block the agent `a1` is gripping, but we must make sure that it is not gripping any. 

The problem is that when checking for the predicates being added/deleted, we check by using Prolog: 

```
P=gripping(a1, _), del(P)=del(gripping(a1, b1).
```

but, this will always be true and the most important part is that while `a1` is part of the resources, `b1` may not and it would make the actions be achievers of one another, even if they may not be when the two actions are actually carried out by two agents.

Basically one would have to compare each argument with the other corresponding one and also make sure that they are grounded correctly.
